internal/blockchain/ethereum/eventstream.go

```diff
@@ -182,7 +182,7 @@ func (s *streamManager) ensureFireFlySubscription(ctx context.Context, instanceP
 		return nil, err
 	}
 
-	subName := fmt.Sprintf("%s_%s", abi.Name, instanceUniqueHash)
+	subName := fmt.Sprintf("%s_%s_%s", namespace, abi.Name, instanceUniqueHash)
 
 	for _, s := range existingSubs {
 		if s.Stream == stream && (s.Name == subName ||
```

Comment:
I think we need some further consideration about pre-existing subscriptions. There are now 3 versions of naming for subscriptions:
1. `{abi.Name}`
2. `{abi.Name}_{instanceHash}`
3. `{abi.Name}_namespace_{instanceHash}`

We're currently honoring 1 & 3. I think we need to honor 2 as well, but I think we should log a warning if the name follows format 1 or 2. We also need to include migration notes indicating you must manually delete these old subscriptions if you want to take advantage of the new functionality for multiple namespaces.